iron-form.html

```diff
@@ -135,27 +158,25 @@
       */
 
       attached: function() {
-        this._nodeObserver = Polymer.dom(this.$.content).observeNodes(
```

Comment:
We should probably use the [new Polymer's helper functions for distributed children detection](https://www.polymer-project.org/2.0/docs/devguide/shadow-dom#slot-apis). @azakus would this work for v0?